chefignore

```diff
@@ -1,101 +0,0 @@
-# Put files/directories that should be ignored in this file when uploading
```

Comment:
You still really want this file. It gets used by chef when uploading to the server to avoid uploading test files you don't need.